docker-compose.build.env

```diff
@@ -3,148 +3,3 @@
 REBUILD=1
 # Set to anything to fetch deps when rebuilding locally in Docker
 FETCH_DEPS=1
-
```

Comment:
We can delete all of the common variables between the `build.env` and the `env` and combine them when the build is desired